scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -40,7 +41,139 @@ import scala.concurrent.Future
  */
 package object bigtable {
 
-  private val DEFAULT_SLEEP_DURATION = Duration.standardMinutes(20);
+  // =======================================================================
+  // Protobuf helpers
+  // =======================================================================
+
+  /** Helper methods for [[Mutation]]. */
+  object Mutations {
+
+    def newSetCell(family: String, column: ByteString, value: ByteString): Mutation =
```

Comment:
There's no default value for `timestampMicros` field in the proto. I guess bigtable service will get 0 as default for `Long` and use that?